### DISCUSSION TITLE
/loja Vanillew

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#1033] [Opinioes]
### Servidor afetado

Vanillew

### Descrição

Permitir que o /loja do Vanillew não necessite que a pessoa vá até a região da loja mas possa ser acessada de qualquer lugar.

**Comment by leogianfagna** [comment]
vou negar pois é o propósito do servidor resgatar as essências vanilla de um servidor. antigamente, todas as lojas eram físicas (e ainda com placa para cada item) e ninguém reclamava.

tirar um sistema desse, também impactaria em:
- menos interação física com outros jogadores
- menos votos na economia rotativa que fica ao lado da loja
- menos visitas em lugares do spawn
